tests/clocks/dff_two_clocks/golden.model.xml

```diff
@@ -2,15 +2,15 @@
 <models>
   <model name="BLOCK">
     <input_ports>
-      <port clock="c1" combinational_sink_ports="o1" name="a"/>
-      <port clock="c2 c1" combinational_sink_ports="o2 o1" name="b"/>
-      <port clock="c2" combinational_sink_ports="o2" name="c"/>
-      <port is_clock="1" name="c1"/>
-      <port is_clock="1" name="c2"/>
+      <port name="a" combinational_sink_ports="o1" clock="c1"/>
+      <port name="b" combinational_sink_ports="o2 o1" clock="c2 c1"/>
+      <port name="c" combinational_sink_ports="o2" clock="c2"/>
+      <port name="c1" is_clock="1"/>
+      <port name="c2" is_clock="1"/>
     </input_ports>
     <output_ports>
-      <port clock="c1" name="o1"/>
-      <port clock="c2" name="o2"/>
+      <port name="o1" clock="c1"/>
+      <port name="o2" clock="c2"/>
```

Comment:
Why do you change the golden model? This is just a cosmetic change, but why is it needed?

Reply:
I think I was trying something and forgot to change it back. Removed the change.